assets/src/application/ui/calendars/dateDisplay/CalendarDateSwitcher/index.tsx

```diff
@@ -12,8 +13,13 @@ const CalendarDateSwitcher: React.FC<CalendarDateSwitcherProps> = React.memo(
 	({ className, displayDate = DisplayStartOrEndDate.start, labels, ...props }) => {
 		const startDate = parseISO(props.startDate) || PLUS_ONE_MONTH;
 		const endDate = parseISO(props.endDate) || PLUS_TWO_MONTHS;
+
+		const footerPast: string | null = propOr<CalendarDateLabels>(null, 'footerPast')(labels);
+		const footerFuture: string | null = propOr<CalendarDateLabels>(null, 'footerFuture')(labels);
+		const footer: string = propOr<CalendarDateLabels>('', 'footer')(labels);
```

Comment:
Why not this `const { footerPast, footerFuture, footer = '' } = labels;`?

```suggestion
	({ className, displayDate = DisplayStartOrEndDate.start, labels = {}, ...props }) => {
		const startDate = parseISO(props.startDate) || PLUS_ONE_MONTH;
		const endDate = parseISO(props.endDate) || PLUS_TWO_MONTHS;

		const { footerPast, footerFuture, footer = '' } = labels;
```

Reply:
in some instances of `CalendarDateSwitcher` there wasn't passed any value to `label` prop, I will make that property required and will go with your suggestion, thx.

Reply:
now that we are passing `getStatusTextLabel` to `CalendarDateSwitcher`, `labels`  won't be empty